classes/submission/maps/Schema.php

```diff
@@ -541,6 +567,9 @@ protected function mapByProperties(array $props, Submission $submission, bool|Co
                 case 'reviewerSuggestions':
                     $output[$prop] = $this->getPropertyReviewerSuggestions($this->reviewerSuggestions);
                     break;
+                case 'issueToBePublished':
```

Comment:
We already have all publications associated with a submission, can we derive this information from there?
Above we have a call `submission->getData('publications')`,
publication has a property `issueId`, `status` and `version`. That should give all needed data to us, shouldn't it?